### PR TITLE
Create voucher header for mobile

### PR DESCRIPTION
**Pull request by GiovannaValadao**
Create voucher header for mobile:

Captura de Tela 2022-02-09 às 19 58 05

### CODE REVIEW

**Comment by raianechristine** [comment]
the header padding is not equal, I had commented [here](https://github.com/GiovannaValadao/clone-home-ifood/pull/2#discussion_r797682605) to make the spacing equal because there is a greater spacing only at the top and the alignment is not correct: 

image

if you get confused you can talk to me ❤️

File: src/components/voucherHeaderMobile/VoucherHeaderMobile.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        </span>
+        <div class="voucher-header-mobile__text">
+          <h4 class="voucher-header-mobile__text-title">Você tem 10 cupons!</h4>
+          <span class="voucher-header-mobile__text-description">
+            Aproveite seus descontos
+          </span>
+        </div>
+        <div class="voucher-header-mobile__icon">
+          <span class="voucher-header-mobile__icon-up">
+            <img src={IconUp} />
```
Comment (raianechristine) [inline src/components/voucherHeaderMobile/VoucherHeaderMobile.jsx:25]: this icon is not visible here,  I think you need to add color to it, or not? 🤔 

image

**Comment by GiovannaValadao** [comment]
@raianechristine True, I remember I changed this, but I don't think it was the commit :(

But I changed it now and it looks like this:


Captura de Tela 2022-02-10 às 20 18 20

File: src/components/voucherHeaderMobile/VoucherHeaderMobile.jsx (hunk 1 above)
Comment (GiovannaValadao) [reply src/components/voucherHeaderMobile/VoucherHeaderMobile.jsx:25]: yes! I have to add the color thanks <3

**Review by raianechristine — APPROVED** [review]

**Pull request merged**